.packit.yaml

```diff
@@ -43,3 +43,17 @@ jobs:
       - centos-stream-9-x86_64
       - rhel-9-x86_64
 
+  - job: copr_build
+    trigger: commit
+    branch: master
+    additional_repos:
+      - "https://download.copr.fedorainfracloud.org/results/%40pki/master/rhel-9-x86_64/"
```

Comment:
There's a trailing backslash here and in `propose_downstream` above for RHEL 9, is this needed? I don't think it's causing any problem, just being OCD :smile: